tbx/project_styleguide/templates/patterns/molecules/streamfield/streamfield-example-service.html

```diff
@@ -0,0 +1,50 @@
+
+<h2 class="heading heading--two  streamfield__heading">Build, consultancy and support from the creators of Wagtail.</h2>
```

Comment:
Hi @lparsons396,

I was just chatting about these example templates with @chris-lawton. They're a bit of a pain to maintain, and I think we should probably find a better solution. I'm going to create a ticket to do this. So for now I think we can remove this.

Cheers,
Helen

Reply:
Sounds good, I'll remove it shortly when I address the rest of the review

Reply:
Sorry @lparsons396 - I mentioned this morning that it would be helpful to add this in before I spoke to Helen about removing them!

Reply:
I've just removed it and pointed the template to the existing streamfield-example-standard.html - this can be redirected as appropriate when the changes in #62 are merged

Reply:
Thanks @lparsons396 - I've just merged #62

Reply:
I've just done this side too so I'll leave it to the FE ticket to link them up